hosting/hostingv4/vmv4.go

```diff
@@ -190,18 +190,24 @@ func (h Hostingv4) CreateVM(vm VMSpec, image DiskImage, version hosting.IPVersio
 // AttachDisk attaches a Disk to a VM, both objects must already exist
 func (h Hostingv4) AttachDisk(vm VM, disk Disk) (VM, Disk, error) {
```

Comment:
I feel like we actually need a position argument. Changing the bootdisk is enough most of the time, but terraform may require changing the order of disks to keep the disk list consistent.